contrib/localnet/deposit-test.sh

```diff
@@ -40,7 +40,7 @@ $TERRAD tx gov deposit 1 10000000uluna --from node1 --keyring-backend test --cha
 
 #
 # loop from 0 to 5 to vote yes
-for i in {0..5}; do
+for i in {0..6}; do
```

Comment:
More magic numbers :P Source from ENV variables, default to desired "sensible default"